src/components/add-products/add-products.js

```diff
@@ -24,10 +40,16 @@ function AddProducts() {
 
   const onSubmit = (event) => {
     setSubmitted(true);
-    if (title && price && categoryId && imageUrl) {
+    if (id === 0 && title && price && categoryId && imageUrl) {
       dispatch(productActions.addProduct(title, price, categoryId, imageUrl));
       dispatch(productActions.getProducts());
```

Comment:
Since this is called in both, it should probably be called outside of the rest of the conditions. The only differing condition is if there is an id or not so,
```
if (!id) {...addProduct} else {...editProduct}
dispatch(...getProducts)
```